ci/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '3.4'
 services:
   weaviate:
-    image: semitechnologies/weaviate:1.16.5
+    image: semitechnologies/weaviate:1.17.0-prealpha-e8b5ccb
```

Comment:
although graphql tests uses just this docker-compose file, it would be good practise to change image in the remaining docker-files as well